src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml.vb

```diff
@@ -21,6 +22,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
             InitializeComponent()
 
             BindToFullSolutionAnalysisOption(Enable_full_solution_analysis, LanguageNames.VisualBasic)
+            BindToOption(Use_editorconfig_compatibility_mode, EditorConfigDocumentOptionsProviderFactory.UseLegacyEditorConfigSupport)
```

Comment:
❓ Is there any place we can add this that either:

1. Is not language-specific, and only appears once
2. Is language-specific, and appears for each language

Reply:
Nothing that we control. The checkboxes are linked -- @JoeRobich did that work because we had this same problem for something else. It's not new.